### PR TITLE
docs(openid-connect): add Authentication through OIDC Relying Party flow description

### PR DESCRIPTION
**Pull request by okaybase**
### What this PR does / why we need it:

add Authentication through OIDC Relying Party flow description

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/openid-connect.md [hunk 1]
```diff
@@ -142,6 +142,41 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
+#### 通过 OIDC 依赖方认证流程进行身份验证
+
+当一个请求在请求头中不包含访问令牌时，也不在会话 cookie 中，
```
Comment (starsz) [inline docs/zh/latest/plugins/openid-connect.md:147]:
```suggestion
当一个请求在请求头或会话 Cookie 中不包含访问令牌时，
```

Comment (okaybase) [reply docs/zh/latest/plugins/openid-connect.md]: okay~ done~ thanks~

File: docs/zh/latest/plugins/openid-connect.md [hunk 2]
```diff
@@ -142,6 +142,41 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
...
+  },
+  "upstream": {
+    "type": "roundrobin",
+    "nodes": {
+      "httpbin.org:443": 1
+    }
+  }
+}'
+```
+
```
Comment (spacewander) [inline docs/zh/latest/plugins/openid-connect.md:179]: Does the zh version miss the translation of "In this case, the plugin can enforce that the access token, the ID token, and the UserInfo object get set in respective configured request headers."?

Comment (okaybase) [reply docs/zh/latest/plugins/openid-connect.md:179]: okay~ done~ thanks~ @spacewander

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**